app/queries/helpers/recipe_query_helper.rb

```diff
@@ -0,0 +1,33 @@
+class RecipeQueryHelper
+
+  def initialize( query=nil )
+    @query = query #Arel.sql( "UNACCENT('%#{ query }%')" )
+  end
+
+  def search
```

Comment:
Please change function name to avoid confusion with scope: search in recipe model